tgs/src/test/cpp/tgs/FeatureSelection/InformationGainCalculatorTest.cpp

```diff
@@ -1,100 +0,0 @@
-/*
```

Comment:
This file is an EXACT duplicate of `SymmetricUncertaintyCalculatorTest.cpp` and the class name and file name match in the other file.  Deleting this file because 
`SymmetricUncertaintyCalculatorTest != InformationGainCalculatorTest`